README.md

```diff
@@ -4,7 +4,7 @@
 
 > This project is about set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features.
 
-![Screenshot](https://github.com/uwadonat/Leaderboard/blob/setup-project/images/Screenshot.png)
+![Screenshot](https://github.com/uwadonat/Leaderboard/blob/hit-api/images/Screenshot.png)
```

Comment:
- [ ] README `live demo` link is broken. Please fix it so that README looks professional. :slightly_smiling_face: